website/Makefile

```diff
@@ -21,3 +22,7 @@ node_modules: package.json
 	@$(NPM) install
 
 install: node_modules
+
+docker-start:
+	$(DOCKER) build -t openbao-docs .
+	$(DOCKER) run -p 3000:3000 --mount "type=bind,source=$(CURDIR),destination=/opt/docusaurus,chown=true,relabel=shared" -it openbao-docs
```

Comment:
Apparently the `chown` and `relabel` attributes only work and are required for rootless Podman.. they don't work for Docker.

Reply:
Fixed.